src/main/java/athena/Athena.java

```diff
@@ -1,45 +1,46 @@
 package athena;
 
 import athena.commands.Command;
+import athena.exceptions.AddException;
+
+import java.util.Scanner;
 
 public class Athena {
-    private static Ui ui;
+    private Ui ui;
+    private Parser parser;
+    private Storage storage;
+    private TaskList taskList;
 
     public Athena() {
         ui = new Ui();
+        parser = new Parser();
+        storage = new Storage("save.csv", ui);
```

Comment:
According to the UG, the file should be data.csv. This was my mistake that I did not fix because it was temporary test code.

Reply:
Ok, I'll amend it